openapi3.yaml

```diff
@@ -61,12 +147,37 @@ components:
       properties:
         message:
           type: string
-    resource:
+    requestBodyForFilter:
+      type: object
+      required:
+        - arrayOfilterProduct
+      properties:
+        arrayOfilterProduct:
+          type: array
+          items:
+            $ref: '#/components/schemas/filterProduct'
+        limit:
+          type: integer
+          minimum: 0
+        skip:
+          type: integer
+          minimum: 0
+    filterProduct:
```

Comment:
I think `productFilter` would be more suitable.

Reply:
agree, change that.